synfig-studio/src/gui/canvasview.cpp

```diff
@@ -1361,6 +1361,16 @@ CanvasView::create_right_toolbar()
 		displaybar->append(*snap_grid);
 	}
 
+	{
+		Gtk::ToggleToolButton *show_ruler = Gtk::manage(new Gtk::ToggleToolButton());
```

Comment:
I suggest to rename it to `toggle_ruler`. `WorkArea::show_hide_ruler()` too.

Reply:
Yes I guess that makes it more unified with the other toggle buttons, will do.
 I'm now working on saving the meta data correctly so that when the file is reopened the button is as left, like the others.